activities/EbookReader.activity/js/toolbar.js

```diff
@@ -22,6 +22,7 @@ var Toolbar = {
 			<toolbar-item id="activity-button" v-bind:title="l10n.stringEbookReaderActivity"></toolbar-item>
 			<toolbar-item isSplitbar="true"></toolbar-item>
 
+			<toolbar-item ref="switchbutton" v-on:clicked="getApp().showContents()" class="toolbutton" id="contents-button" v-bind:title="l10n.stringContents" v-if="getApp().currentView == getReader()"></toolbar-item>
```

Comment:
Not the better place to add it. Will be better after the next button.
Warning: `ref` property should be unique

Reply:
Fixed.